plugins/acss-kit/commands/utility-tune.md

```
@@ -0,0 +1,30 @@
+description: Adjust the utility-class tokens (spacing baseline, breakpoints, family enables) via natural language
+argument-hint: <natural-language description>
+allowed-tools: Read, Glob, Grep, Write, Edit, Bash, AskUserQuestion
+---
+
+Adjust `utilities.tokens.json` based on a natural-language request, regenerate the bundle, and validate the result.
+
+Follow the `/utility-tune` section of `${CLAUDE_PLUGIN_ROOT}/skills/utilities/SKILL.md`.
+
+**Examples:**
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)